packages/dds/merge-tree/package.json

```diff
@@ -109,6 +109,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"InterfaceDeclaration_SerializedAttributionCollection": {
+				"backCompat": false
```

Comment:
Should this change be based off `next`, or is this fine because the type is marked `@alpha`/because the breaking change is adding a new type to a union and we don't guarantee such changes to be released in major version bumps?

Reply:
yeah, this type won't ever actually be exposed (it's marked internal, not alpha). we need to ensure forward compat since it's a type that ends up written to disk, but extending the set of possible keys satisfies that. it's also true that local attribution keys should never make their ways to summaries (summarizer client should never have local edits), so technically the change to AttributionKey shouldn't affect the representation of this type at all, ever. (I have an assert for that in my merge-tree change which leverages this)